### PR TITLE
Add 3.3V and 5V output control

### PR DESCRIPTION
**Pull request by 21km43**
I added these commands, and already tested on WCH-LinkE.
```
> wlink set-power --help 
Enable or disable power output

Usage: wlink set-power [OPTIONS] <COMMAND>

Commands:
  enable3v3   Enable 3.3V output
  disable3v3  Disable 3.3V output
  enable5v    Enable 5V output
  disable5v   Disable 5V output
  help        Print this message or the help of the given subcommand(s)
```

example
```
> wlink set-power enable3v3
13:31:46 [INFO] Connected to WCH-Link v2.12(v32) (WCH-LinkE-CH32V305)
13:31:46 [INFO] Enable 3.3V Output

> wlink set-power disable3v3
13:31:52 [INFO] Connected to WCH-Link v2.12(v32) (WCH-LinkE-CH32V305)
13:31:52 [INFO] Disable 3.3V Output

> wlink set-power enable5v
13:32:03 [INFO] Connected to WCH-Link v2.12(v32) (WCH-LinkE-CH32V305)
13:32:03 [INFO] Enable 5V Output

> wlink set-power disable5v
13:32:07 [INFO] Connected to WCH-Link v2.12(v32) (WCH-LinkE-CH32V305)
13:32:07 [INFO] Disable 5V Output
```

When merge this PR, I recommend that squash and merge.

### CODE REVIEW

**Comment by 21km43** [comment]
command format is not beautiful...
```
  enable3-v3   Enable 3.3V output
  disable3-v3  Disable 3.3V output
  enable5-v    Enable 5V output
  disable5-v   Disable 5V output
```

**Comment by 21km43** [comment]
At least I confirmed that my program works.

**Comment by 21km43** [comment]
If someone have a idea of command fotmat, feel free to comment to this PR

File: src/main.rs [hunk 1]
```diff
@@ -148,6 +148,14 @@ enum Commands {
...
     /// List probes
     List {},
+    /// Enable 3.3V output
+    Enable3v3 {},
+    /// Disable 3.3V output
+    Disable3v3 {},
+    /// Enable 5V output
+    Enable5v {},
+    /// Disable 5V output
+    Disable5v {},
```
Comment (andelf) [inline src/main.rs:158]: Better make them grouped into a `SetPower(commands::SetPower)` sub-command.

Comment (21km43) [reply src/main.rs]: Thank you for review.
I grouped then into sub-command.

**Review by andelf — APPROVED** [review]
Looks great! Approved.

**Pull request merged**